### DISCUSSION TITLE
Feature-suggestion - Filter that overwrites other Filters (except you tell it not to)

### DISCUSSION THREAD

**Discussion by Husky110** [filamentphp/filament#6927] [Showcase]
During one of my help-requests I established a feature where a ToggleFilter is able to overwrite any other filter set on a table. I think this would make a great addition to filament in general, since I could see the pattern "give me everything that has parameter X set to exclusively A, only if I tell it otherwise". I use this pattern for "quick-filters" where I don't want the user to be bothered by any other filter.  
Since this filter kills all other filters I named it "KillerFilter"... :D Here is it's gist: https://gist.github.com/Husky110/080711a2a17fa3647d0150fe41b22152  
Maybe in the main implementation there could be a function like "exclusive" which accepts an array of others filters it should not overwrite or it's a special Toggle-Filter - I don't know.  
Just wanted to spread the idea of "exclusive filters". :)

**Comment by danharrin** [comment]
How about using a single SelectFilter for all filters that you can toggle between?

**Reply by Husky110** [reply]
That's what I first did, but those work additive, which is not what I want to achieve here. In my scenario we're talking about business-orders. I used the KillerFilter to quickly show all orders that are currently open or all orders that are in shipping or all orders that have been done. The status itself is stored in a "status_id"-column which has several in-between-status. In this case it's all about having one toggle to say "give me all orders that are in shipping and discard all previous set filters".

**Reply by Husky110** [reply]
Plus: The user does not have to discard filters he/she set beforehand.
